server/container/core/src/test/java/org/apache/james/server/core/MimeMessageTest.java

```diff
@@ -311,9 +311,9 @@ public void testGetLineCount() throws Exception {
     public void testMessageCloningViaCoW() throws Exception {
```

Comment:
at a minimum the test name is misleading since there is no more COW :) you can probably drop this test

Reply:
I hesitated ;-)